src/platform/CMakeLists.txt

```diff
@@ -132,5 +132,8 @@ if(QUIC_TLS STREQUAL "openssl" OR QUIC_TLS STREQUAL "openssl3")
         target_link_libraries(platform PUBLIC "-framework CoreFoundation" "-framework Security")
     endif()
 elseif(QUIC_TLS STREQUAL "schannel")
-    target_link_libraries(platform PUBLIC secur32 onecore)
+    target_link_libraries(platform PUBLIC secur32)
```

Comment:
I think this change broke normal Windows builds:
```
platform.lib(tls_schannel.obj) : error LNK2019: unresolved external symbol _QueryContextAttributesExW@16 referenced in function _CxPlatTlsParamGet [D:\a\msquic\msquic\build\windows\x86_schannel\src\bin\msquic.vcxproj]
```

Reply:
reverted this part

Reply:
`onecore` must not be linked in GAMECORE builds because linking 2 umbrella libs (xgameplatform and onecore) is not supported. Changed it to special case gamecore builds